spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/EventProcessorProperties.java

```diff
@@ -301,5 +306,53 @@ public String getSequencingPolicy() {
         public void setSequencingPolicy(String sequencingPolicy) {
             this.sequencingPolicy = sequencingPolicy;
         }
+
+        /**
+         * Retrieves the AutoConfiguration settings for the sequenced dead letter queue settings.
+         *
+         * @return the AutoConfiguration settings for the sequenced dead letter queue settings
+         */
+        public Dlq getDlq() {
+            return dlq;
+        }
+
+        /**
+         * Defines the AutoConfiguration settings for the sequenced dead letter queue.
+         *
+         * @param dlq the sequenced dead letter queue settings settings for the token store.
+         */
+        public void setDlq(Dlq dlq) {
+            this.dlq = dlq;
+        }
+    }
+
+    public static class Dlq {
+
+        /**
+         * Enables creation and configuring a {@link org.axonframework.messaging.deadletter.SequencedDeadLetterQueue} by
+         * using the available {@link org.axonframework.messaging.deadletter.EventProcessingSdlqFactory}. Defaults to
```

Comment:
I think this documentation isn't correct anymore, as the PR does not contain a `EventProcessingSdlqFactory`.

Reply:
Documentation was changed.